numba_rvsdg/core/datastructures/basic_block.py

```diff
@@ -43,34 +96,91 @@ def declare_backedge(self, target: str) -> "BasicBlock":
 
     def replace_jump_targets(self, jump_targets: Tuple) -> "BasicBlock":
         """Replaces jump targets of this block by the given tuple.
+
+        This method replaces the jump targets of  the current BasicBlock.
         The provided jump targets must be in the same order as their
         intended original replacements.
 
         Note that replacing jump targets will not replace the backedge
         tuple, so replacement for any jump targets that is declared as
-        a backedge needs to be updated separately using replace_backedges"""
+        a backedge needs to be updated separately using replace_backedges
+
+        Parameters
+        ----------
+        jump_targets: Tuple
+            The new jump target tuple. Must be ordered.
+
+        Returns
+        -------
+        result: BasicBlock
```

Comment:
```suggestion
        basic_block: BasicBlock
```

This notation was used above too, let's keep it consistent.